crates/vault-registry/src/lib.rs

```diff
@@ -721,6 +723,7 @@ impl<T: Config> Pallet<T> {
         let vault = Vault::new(vault_id.clone(), public_key);
         Self::insert_vault(&vault_id, vault);
 
+        // SBP M1 Impact of this potential failure on previously updated data?
```

Comment:
Are the remaining comments in this file related to our usage of `transactional`?

Reply:
Right, I didn't verify if all paths leading to the use of those functions were using `transactional`.